impeller/display_list/canvas.cc

```diff
@@ -1086,6 +1123,42 @@ void Canvas::SaveLayer(const Paint& paint,
         transform_stack_.back().transform.HasTranslation()
             ? Entity::RenderingMode::kSubpassPrependSnapshotTransform
             : Entity::RenderingMode::kSubpassAppendSnapshotTransform);
+
+    if (will_cache_backdrop_texture) {
+      FML_DCHECK(backdrop_data);
+      // If all filters on the shared backdrop layer are equal, process the
+      // layer once.
+      if (backdrop_data->all_filters_equal &&
+          !backdrop_data->shared_filter_snapshot.has_value()) {
+        // TODO(jonahwilliams): compute minimum input hint.
```

Comment:
This needs an issue.

Reply:
ahh forgot to update it, I had filled https://github.com/flutter/flutter/issues/157110

Reply:
done